packages/shared/src/db/queries.ts

```diff
@@ -153,6 +153,32 @@ export const getGroupPreviews = createReadQuery(
   ['groups']
 );
 
+export const getJoinedGroupsCount = createReadQuery(
+  'getJoinedGroupCount',
+  async (ctx: QueryCtx) => {
+    const joinedGroups = await ctx.db.query.groups.findMany({
+      where: eq($groups.currentUserIsMember, true),
```

Comment:
It'd be more efficient to just grab the count directly, rather than returning the full results array -- check out `getContactsCount`